ts/features/messages/store/reducers/__tests__/detailsById.test.ts

```diff
@@ -69,6 +73,48 @@ describe("detailsById reducer", () => {
       });
     });
   });
+  describe("AAR flow related logic", () => {
```

Comment:
We are testing `populateStoresWithEphemeralAarMessageData` so why not using it in the describe?